mcs/resource_mcs_db_cluster_with_shards.go

```diff
@@ -97,6 +100,27 @@ func resourceDatabaseClusterWithShards() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"disk_autoexpand": {
```

Comment:
Don't we need the same for wal volumes?

Reply:
For wal volumes we already have it, here: https://github.com/MailRuCloudSolutions/terraform-provider-mcs/blob/d5d9947e47ad20b897c7bc8b04eafaedc3800b42/mcs/resource_mcs_db_cluster_with_shards.go#L174